bokeh/server/views/static_handler.py

```diff
@@ -68,7 +68,7 @@ def append_version(cls, path):
             return path
         else:
```

Comment:
There are some out of date comments in this file, as long as you are tidying up they could be removed/improved?